docs/developer/decision-records/2022-07-01-get-claims/README.md

```diff
@@ -0,0 +1,40 @@
+# Identity Hub - Get claims
+
+This document explains how we will get the claims of a participant, in order to apply access policies.
+
+## Context
+
+When an EDC participant receives an IDS request, it must verify the caller's identity and apply access policies for the caller.
+To apply policies, it needs to get the claims of the participant.
+This document explains the mechanism to get the claims from IdentityHub.
+
+## What is a claim
+
+A claim is a statement about a subject. For example, a claim could be `ParticipantA` is in `region` `eu`.
+See [this documentation](https://www.w3.org/TR/vc-data-model/#claims) for more details.
+
+## Get claims
+
+This document focuses on getting the claims of a Participant, assuming the participant's identity is already verified.
+The participant will use the claims to apply the access policy.  
+Each participant has an IdentityHub instance, exposing an endpoint to get its [Verifiable Credentials](https://www.w3.org/TR/vc-data-model/).
+The IdentityHub endpoint returns a list of JWS containing a Verifiable Credential in their payload.
+(see [example Verifiable Credential (as JWT)](https://www.w3.org/TR/vc-data-model/#example-usage-of-the-credentialsubject-property)).
+The VerifiableCredentials contain [claims](https://www.w3.org/TR/vc-data-model/#claims) that can be extracted.
+
+![Apply policy flow](apply-policies-flow.png)
+
+Let's focus on the `Get claims` box.
+`Participant B` wants to get the claims of `Participant A`, to apply access policy.
+For example, Participant A could have the policy that Participant B should be claimed as a `participantOf`
```

Comment:
```suggestion
For example, `Participant A` could have the policy that `Participant B` should be claimed as a `participantOf`
```
in addition: if `B` wants to get `A`'s claims, it is `B` who applies its access policy. Therefor it would likely be `A` who is a participant of `dataspaceA`, right?

Reply:
Good catch! I mixed the 2 by mistake.

I fixed it and I also renamed the `dataspaceA` to `dataspaceC`, to avoid confusion about the dataspace belonging to A. As the policy could be about any dataspace, not something belonging to A.